eskip/string.go

```diff
@@ -80,6 +80,22 @@ func argsString(args []interface{}) string {
 	return strings.Join(sargs, ", ")
 }
 
+func sortedKeys[V string | []string](m map[string]V) []string {
```

Comment:
`func sortedKeys[V any](m map[string]V) []string `

Reply:
we decided to go with `interface{}` instead of `any` since we override `any` in [eskip/parser.go](https://github.com/zalando/skipper/blob/master/eskip/parser.go#L47)